vgregion.se,"HTTPSConnectionPool(host='www.vgregion.se', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.vgregion.se', port=443): Read timed out. (read timeout=2)"
vhodne-uverejneni.cz,"HTTPSConnectionPool(host='www.vhodne-uverejneni.cz', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.vhodne-uverejneni.cz', port=443): Read timed out. (read timeout=2)"
vietnamplus.vn,"HTTPConnectionPool(host='www.vietnamplus.vn', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff347e43850>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))","HTTPConnectionPool(host='www.vietnamplus.vn', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7ff3462715b0>, 'Connection to www.vietnamplus.vn timed out. (connect timeout=2)'))"
vil.ee,"HTTPSConnectionPool(host='vil.ee', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff346271f40>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='vil.ee', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff346787700>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
villalivet.se,"HTTPSConnectionPool(host='villalivet.se', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='villalivet.se', port=443): Read timed out. (read timeout=2)"
vilniausgalerija.lt,"HTTPSConnectionPool(host='vilniausgalerija.lt', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='vilniausgalerija.lt', port=443): Read timed out. (read timeout=2)"
vilnius.lt,"HTTPSConnectionPool(host='vilnius.lt', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='vilnius.lt', port=443): Read timed out. (read timeout=2)"
vinted.es,"HTTPSConnectionPool(host='www.vinted.es', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.vinted.es', port=443): Read timed out. (read timeout=2)"
violet.vn,"HTTPSConnectionPool(host='violet.vn', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))","HTTPSConnectionPool(host='violet.vn', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))"
violity.com,"HTTPSConnectionPool(host='violity.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='violity.com', port=443): Read timed out. (read timeout=2)"
visitportugal.com,"HTTPSConnectionPool(host='visitportugal.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1129)')))","HTTPSConnectionPool(host='visitportugal.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1129)')))"
viss.lt,"HTTPSConnectionPool(host='viss.lt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1129)')))","HTTPConnectionPool(host='viss.lt', port=80): Read timed out. (read timeout=2)"
viss.lv,"HTTPSConnectionPool(host='viss.lv', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1129)')))","HTTPConnectionPool(host='viss.lv', port=80): Read timed out. (read timeout=2)"
vital.hu,"HTTPSConnectionPool(host='vital.hu', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='vital.hu', port=80): Read timed out. (read timeout=2)"
vksndtc.gov.vn,"HTTPSConnectionPool(host='vksndtc.gov.vn', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))","HTTPSConnectionPool(host='vksndtc.gov.vn', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))"
vlaamsparlement.be,"HTTPSConnectionPool(host='vlaamsparlement.be', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff3488e8a30>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='vlaamsparlement.be', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff348a115b0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
vlaamswoordenboek.be,"HTTPSConnectionPool(host='vlaamswoordenboek.be', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1129)')))","HTTPConnectionPool(host='vlaamswoordenboek.be', port=80): Read timed out. (read timeout=2)"
vladi.si,"HTTPSConnectionPool(host='vladi.si', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff347a67dc0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='vladi.si', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff342420cd0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
vloerverwarmingen-prijs.be,"HTTPSConnectionPool(host='vloerverwarmingen-prijs.be', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff34a28f580>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='vloerverwarmingen-prijs.be', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff34993cb20>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
vmd.gov.lv,"HTTPSConnectionPool(host='vmd.gov.lv', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ff34a257610>, 'Connection to vmd.gov.lv timed out. (connect timeout=2)'))","HTTPConnectionPool(host='vmd.gov.lv', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7ff349f336a0>, 'Connection to vmd.gov.lv timed out. (connect timeout=2)'))"
vnanet.vn,"HTTPSConnectionPool(host='vnanet.vn', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ff348cb2ca0>, 'Connection to vnanet.vn timed out. (connect timeout=2)'))","HTTPConnectionPool(host='vnanet.vn', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7ff3419a6040>, 'Connection to vnanet.vn timed out. (connect timeout=2)'))"
vodacom.co.za,"HTTPSConnectionPool(host='vodacom.co.za', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'vodacom.co.za' doesn't match either of 'www.vodacom.co.za', 'web.vodacom.co.za'"")))","HTTPSConnectionPool(host='vodacom.co.za', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'vodacom.co.za' doesn't match either of 'www.vodacom.co.za', 'web.vodacom.co.za'"")))"
vodafone.co.nz,"HTTPSConnectionPool(host='www.vodafone.co.nz', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.vodafone.co.nz', port=443): Read timed out. (read timeout=2)"
vodafone.es,"HTTPSConnectionPool(host='vodafone.es', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1129)')))","HTTPSConnectionPool(host='www.vodafone.es', port=443): Max retries exceeded with url: /c/particulares/es/ (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1129)')))"
vodafone.ua,"HTTPSConnectionPool(host='vodafone.ua', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1129)')))","HTTPSConnectionPool(host='vodafone.ua', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1129)')))"
voici.fr,"HTTPSConnectionPool(host='voici.fr', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ff34b136670>, 'Connection to voici.fr timed out. (connect timeout=2)'))","HTTPSConnectionPool(host='www.voici.fr', port=443): Read timed out. (read timeout=2)"
vpvb.gov.lv,"HTTPSConnectionPool(host='vpvb.gov.lv', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff34133b910>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='vpvb.gov.lv', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff347b50eb0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
vru.ac.th,"HTTPSConnectionPool(host='vru.ac.th', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ff342320520>, 'Connection to vru.ac.th timed out. (connect timeout=2)'))","HTTPConnectionPool(host='vru.ac.th', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7ff345aab820>, 'Connection to vru.ac.th timed out. (connect timeout=2)'))"
vsb.cz,"HTTPSConnectionPool(host='vsb.cz', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff34a762b20>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='vsb.cz', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff343e1f370>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
vtv.gob.ve,"HTTPSConnectionPool(host='vtv.gob.ve', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))","HTTPSConnectionPool(host='www.vtv.gob.ve', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))"
vu.edu.pk,"HTTPSConnectionPool(host='vu.edu.pk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))","HTTPSConnectionPool(host='vu.edu.pk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))"
vuka.hr,"HTTPSConnectionPool(host='vuka.hr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff3496d3550>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='vuka.hr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff347f165b0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
vvkt.lt,"HTTPSConnectionPool(host='vvkt.lt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))","HTTPSConnectionPool(host='vvkt.lt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))"
vylety-zabava.cz,"HTTPSConnectionPool(host='vylety-zabava.cz', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='vylety-zabava.cz', port=443): Read timed out. (read timeout=2)"
waa2.co.za,"HTTPSConnectionPool(host='www.waa2.co.za', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))","HTTPSConnectionPool(host='www.waa2.co.za', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))"
walaw.press,"HTTPSConnectionPool(host='walaw.press', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='walaw.press', port=443): Read timed out. (read timeout=2)"
waroengsipiet.nl,"HTTPSConnectionPool(host='waroengsipiet.nl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff348ad7430>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))","HTTPConnectionPool(host='waroengsipiet.nl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff340f7c310>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))"
wartaekonomi.co.id,"HTTPSConnectionPool(host='wartaekonomi.co.id', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='wartaekonomi.co.id', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ff349579040>, 'Connection to wartaekonomi.co.id timed out. (connect timeout=2)'))"
waseda.jp,"HTTPSConnectionPool(host='waseda.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff349f6c5b0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='waseda.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff34941ec40>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
wasfetmama.com,"HTTPSConnectionPool(host='wasfetmama.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='wasfetmama.com', port=443): Read timed out. (read timeout=2)"
washingtonpost.com,"HTTPSConnectionPool(host='www.washingtonpost.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.washingtonpost.com', port=443): Read timed out. (read timeout=2)"
watsons.co.id,"HTTPSConnectionPool(host='watsons.co.id', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='watsons.co.id', port=80): Read timed out. (read timeout=2)"
watsons.co.th,"HTTPSConnectionPool(host='watsons.co.th', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='watsons.co.th', port=80): Read timed out. (read timeout=2)"
watsons.com.my,"HTTPSConnectionPool(host='watsons.com.my', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='watsons.com.my', port=80): Read timed out. (read timeout=2)"
watsons.com.sg,"HTTPSConnectionPool(host='watsons.com.sg', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='watsons.com.sg', port=80): Read timed out. (read timeout=2)"
watsons.ua,"HTTPSConnectionPool(host='watsons.ua', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='watsons.ua', port=80): Read timed out. (read timeout=2)"
waykanankab.go.id,"HTTPSConnectionPool(host='waykanankab.go.id', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ff34959cdf0>, 'Connection to waykanankab.go.id timed out. (connect timeout=2)'))","HTTPConnectionPool(host='waykanankab.go.id', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7ff346baca30>, 'Connection to waykanankab.go.id timed out. (connect timeout=2)'))"
wbc.poznan.pl,"HTTPSConnectionPool(host='wbc.poznan.pl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'wbc.poznan.pl' doesn't match 'www.wbc.poznan.pl'"")))","HTTPSConnectionPool(host='wbc.poznan.pl', port=443): Max retries exceeded with url: /dlibra (Caused by SSLError(SSLCertVerificationError(""hostname 'wbc.poznan.pl' doesn't match 'www.wbc.poznan.pl'"")))"
wbpublibnet.gov.in,"HTTPSConnectionPool(host='wbpublibnet.gov.in', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7ff34c2868e0>, 'Connection to wbpublibnet.gov.in timed out. (connect timeout=2)'))","HTTPConnectionPool(host='wbpublibnet.gov.in', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7ff340cb3ac0>, 'Connection to wbpublibnet.gov.in timed out. (connect timeout=2)'))"
webry.info,"HTTPSConnectionPool(host='webry.info', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff3423a7550>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='webry.info', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff341107e80>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
wellcare.vn,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
wendydansenfotografie.nl,"HTTPSConnectionPool(host='wendydansenfotografie.nl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ff340b946d0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))","HTTPConnectionPool(host='wendydansenfotografie.nl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff3441bffa0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))"
what-this.com,"HTTPSConnectionPool(host='what-this.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='what-this.com', port=443): Read timed out. (read timeout=2)"
wiki.tn,"HTTPSConnectionPool(host='wiki.tn', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'wiki.tn' doesn't match 'www.wiki.tn'"")))","HTTPSConnectionPool(host='wiki.tn', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'wiki.tn' doesn't match 'www.wiki.tn'"")))"
wikicachlam.com,"HTTPSConnectionPool(host='wikicachlam.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))","HTTPSConnectionPool(host='wikicachlam.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))"
wikionline.site,"HTTPSConnectionPool(host='wikionline.site', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='wikionline.site', port=443): Read timed out. (read timeout=2)"
